build.gradle

```diff
@@ -67,8 +67,13 @@ configurations {
 
 test {
     testLogging {
-        events = ["passed", "failed", "skipped"]
-        showStandardStreams = false
+        onOutput { descriptor, event ->
+            if (project.hasProperty('logOutput')) {
```

Comment:
This change along with the fix (https://github.com/pravega/flink-connectors/commit/c872b3c0d7052b4f815275770145ccc7bbcffd30) to travis config will enable logging when Travis builds the project. On standalone mode, we need to pass the `-PlogOutput` property to the gradle command if we want to see the build/test output in console `./gradlew test -PlogOutput`